list-pr-files/README.md

```diff
@@ -0,0 +1 @@
+# list-pr-files
```

Comment:
Maybe add a TODO that this README needs updating? Although it _is_ fairly obvious it should be :D